### PR TITLE
Update install instructions

### LINKED ISSUE #49: Update install instructions
With your updated install for the package, is [this](https://github.com/Imageomics/kabr-tools/blob/19b9c7908ba20c544215452a99335d08e7139e05/README.md?plain=1#L103) and everything above from line 91 necessary? I do think including a note about the package install containing a workaround for their outdated `setup.py` is still a good thing to include in the README.

_Originally posted by @egrace479 in https://github.com/Imageomics/kabr-tools/pull/40#discussion_r1777400765_

[Full discussion](https://github.com/Imageomics/kabr-tools/pull/40#discussion_r1777400765). We want to update with some clarity on the special aspects and some of the potential challenges and ways to address them.

### PR DESCRIPTION
**Pull request by zhong-al**
Resolves #49 
- Removes unnecessary instructions
- Adds install notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ The KABR tools used in this process can be installed with:
 ```
 pip install git+https://github.com/Imageomics/kabr-tools
 ```
+
+**Notes:**
+ - detectron2 requires Linux or MacOS
+ - If building detectron2's wheel fails, loading a different gnu module may help (`module load gnu/11.2.0`)
+ - If ModuleNotFoundError: No module named 'torch' appears, try `pip install torch torchvision`
```
Comment (egrace479) [inline README.md:19]:
```suggestion
 - detectron2 requires Linux or MacOS.
 - If building detectron2's wheel fails, loading a different gnu module may help (`module load gnu/11.2.0`).
 - If `ModuleNotFoundError: No module named 'torch'` appears, try `pip install torch torchvision` in your environment and try installing the package again.
```
Also add the note about the workaround for SlowFast?

Comment (zhong-al) [reply README.md:19]: "the package" might be confusing. Can I replace it with KABR tools?

Comment (egrace479) [reply README.md:19]: Yes, please do!

File: README.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ pip install git+https://github.com/Imageomics/kabr-tools
 ```
 
 **Notes:**
- - detectron2 requires Linux or MacOS
- - If building detectron2's wheel fails, loading a different gnu module may help (`module load gnu/11.2.0`)
- - If ModuleNotFoundError: No module named 'torch' appears, try `pip install torch torchvision`
+ - detectron2 requires Linux or MacOS.
+ - If building detectron2's wheel fails, loading a different gnu module may help (`module load gnu/11.2.0`).
+ - If `ModuleNotFoundError: No module named 'torch'` appears, try `pip install torch torchvision` in your environment and try installing `kabr_tools` again.
  - SlowFast's setup.py is outdated; our workaround is `pip install git+https://github.com/zhong-al/SlowFast@797a6f3ae81c49019d006296f1e0f84f431dc356`
```
Comment (egrace479) [inline README.md:20]:
```suggestion
 - SlowFast's setup.py is outdated; our workaround is `pip install git+https://github.com/zhong-al/SlowFast@797a6f3ae81c49019d006296f1e0f84f431dc356`, which is included in when installing `kabr_tools`.
```

File: README.md [hunk 3]
```diff
@@ -88,26 +95,8 @@ You can use the [KABR model](https://huggingface.co/imageomics/x3d-kabr-kinetics
...
-Installing detectron2 and pytorchvideo can be tricky. This should work:
-```
-python -m pip install git+https://github.com/facebookresearch/detectron2.git@2a420edb307c9bdf640f036d3b196bed474b8593
-python -m pip install git+https://github.com/facebookresearch/pytorchvideo.git@1fadaef40dd393ca09680f55582399f4679fc9b7
-```
-
-[SlowFast](https://github.com/facebookresearch/SlowFast)'s `setup.py` is outdated, so a workaround is:
-
-```
-python -m pip install git+https://github.com/zhong-al/SlowFast@797a6f3ae81c49019d006296f1e0f84f431dc356
```
Comment (hlapp) [inline README.md:100]: If this is an "official" part of KABR tools, then perhaps it's better to host the fork under the imageomics org?

File: README.md [hunk 4]
```diff
@@ -12,6 +12,13 @@ The KABR tools used in this process can be installed with:
 ```
 pip install git+https://github.com/Imageomics/kabr-tools
 ```
+
+**Notes:**
+ - detectron2 requires Linux or MacOS.
+ - If building detectron2's wheel fails, loading a different gnu module may help (`module load gnu/11.2.0`).
+ - If `ModuleNotFoundError: No module named 'torch'` appears, try `pip install torch torchvision` in your environment and try installing `kabr_tools` again.
+ - SlowFast's setup.py is outdated; our workaround is `pip install git+https://github.com/zhong-al/SlowFast@797a6f3ae81c49019d006296f1e0f84f431dc356`, which is included in when installing `kabr_tools`.
```
Comment (hlapp) [inline README.md:20]: If this is an "official" part of KABR tools, then perhaps it's better to host the fork under the imageomics org?

Comment (egrace479) [reply README.md:20]: This is @zhong-al's fix to the [SlowFast dependency issue](https://github.com/facebookresearch/SlowFast/issues/721). She submitted a [PR](https://github.com/facebookresearch/SlowFast/pull/722) to fix it, but they have not been responsive.

Comment (hlapp) [reply README.md:20]: I understand that, which is why I made the suggestion.

Comment (zhong-al) [reply README.md:20]: Should I transfer ownership of the fork to Imageomics?

Comment (egrace479) [reply README.md:20]: I misunderstood. Yes, @zhong-al, you can.

Comment (zhong-al) [reply README.md]: Transferred & replaced zhong-al with Imageomics

**Review by egrace479 — APPROVED** [review]

**Pull request merged**